Project/Assets/Prairie/Framework/Script/Interaction/AnnotationInteraction.cs

```diff
@@ -169,7 +179,11 @@ void Update()
                 Cursor.visible = false;
                 Cursor.lockState = CursorLockMode.Locked;
                 
-				this.SetPlayerIsFrozen (false);
+				FirstPersonInteractor player = this.GetPlayer ();
```

Comment:
I believe the reason its pulled it out like this,  is that sometimes we just want to freeze the player,  and sometimes  we want to do both.